src/components/MTableHeader/index.js

```diff
@@ -168,60 +270,22 @@ export function MTableHeader({ onColumnResized, ...props }) {
                   {...provided.draggableProps}
                   {...provided.dragHandleProps}
                   style={
-                    snapshot.isDragging ? provided.draggableProps.style : {}
+                    snapshot.isDragging
+                      ? {
+                          ...provided.draggableProps.style,
+                          position: 'static'
+                        }
+                      : { position: 'relative', minWidth: 0, display: 'flex' }
                   }
                 >
                   {columnDef.title}
+                  {renderSortButton(columnDef)}
```

Comment:
I am thinki if this could be its own react component to avoid remounts?

Reply:
That's done - I've left in the same file